### PR TITLE
Tickets/dm 6367

### PR DESCRIPTION
**Pull request by MelissaGraham**
It's time to merge to master.

### CODE REVIEW

File: DMTN-049.tex [hunk 1]
```diff
@@ -3,315 +3,628 @@
...
+This is a \textit{\textbf{living document}} that will evolve over time to incorporate input from the science community, which has a considerable wealth of expertise in generating photo-$z$ catalogs and will be the primary users of the photo-$z$ data product.
+
+
+\section{Roadmap and Timeline}\label{sec:time}
+
+All items in this timeline reference \textbf{\textit{actions taken by DM}}.
+
+{\bf 2021-02-01:} Write a summary of this proposed timeline. Advertise it broadly.
+
+{\bf 2021-02 to 04:} Hold three LSST Photo-$z$ Virtual Forums, one per month, which focus on ingesting input from the science community regarding the proposed evaluation criteria (\S~\ref{sec:eval}). Advertise the forums broadly.
```
Comment (leannep) [inline DMTN-049.tex]: ISO date format writes a range as 2021-02/04

File: DMTN-049.tex [hunk 2]
```diff
@@ -3,315 +3,628 @@
...
+
+{\bf 2021-05-01:} Write a call for "Letters of Recommendation" from the science community. Advertise it broadly.
+
+{\bf 2021-05 to 08:} Hold four LSST Photo-$z$ Virtual Forums, one per month, which focus on supporting the science community's efforts to prepare "Letters of Recommendation" (\S~\ref{sec:lor}). Advertise the forums broadly.
```
Comment (leannep) [inline DMTN-049.tex]: ISO date format writes a range as 2021-05/08

File: DMTN-049.tex [hunk 3]
```diff
@@ -3,315 +3,628 @@
...
+\item could include or reference {\it quantitative} analyses of simulated LSST data, or data from other facilities
+\item will be openly shared in the Rubin community
 \end{itemize}
 
-Reconciling the photo-$z$ option that best suits internal needs with external might be quite difficult and furthermore, unnecessary. Jeff Newman from DESC-PZ commented that the internal-use photo-$z$ \textit{``will almost certainly be different/simpler photo-$z$s than the database ones.''}
+It is understood that "Letters of Recommendation" from members of the science community who are future users of the {\tt Object} catalog photo-$z$, but who are not developers of photo-$z$ estimators, would focus on recommending third-party software based on their science needs, and might not be able to fully address the technical considerations.
+
+However, "Letters of Recommendation" from teams who also plan to support the validation and implementation of one or more estimators during the commissioning-era Photo-$z$ Validation Cooperative (and beyond, to operations) should also please address the following technical considerations as best they can (summarized from Section \ref{sec:dmcon}):
```
Comment (leannep) [inline DMTN-049.tex]: "should also address" (delete 'please')

File: DMTN-049.tex [hunk 4]
```diff
@@ -3,315 +3,628 @@
...
+
+\begin{itemize}
+\item scalability to $\sim$ 37 billion {\tt Objects}
+\item input measured quantities not currently in the DR {\tt Object} schema
+\item estimator outputs not currently in the DR {\tt Object} schema
+\item list of auxiliary data (and estimate storage requirements)
```
Comment (leannep) [inline DMTN-049.tex]: "and estimated storage requirements" or  "and estimate of storage requirements"

File: DMTN-049.tex [hunk 5]
```diff
@@ -3,315 +3,628 @@
...
+\item community support for training, validation, and maintenance
+\item implementation language and adaptation to a Python harness
```
Comment (leannep) [inline DMTN-049.tex]: "and adaptation to a Python harness if needed". Code in python will not need adaptation

File: DMTN-049.tex [hunk 6]
```diff
@@ -3,315 +3,628 @@
...
+{\bf Wide Area Imaging --} 
+Some photo-$z$ methods have requirements other than spec-$z$ fields: e.g., \citet{2019MNRAS.483.2801S} use clustering information to obtain photo-$z$ and this requires wider, shallower field coverage and not a single deep pointing like a spec-$z$ field would have. 
+This wider area would also serve to reduce cosmic variance in the training set ($\sim$100 square degrees would serve to average out the variance).
 
-It is unclear how much additional computation resources would be available in the DR Processing budget for photo-$z$ compression, let alone running multiple estimators
+{\bf Data Previews --}
```
Comment (leannep) [inline DMTN-049.tex]: Preliminary LSST Data

File: DMTN-049.tex [hunk 7]
```diff
@@ -3,315 +3,628 @@
...
+For the community to participate in the training or calibration of a photo-$z$ estimator prior to a data release, it will probably be necessary to release a small ($\sim10\%$) but representative "DR preview" in advance of each data release during Operations.
```
Comment (leannep) [inline DMTN-049.tex]: "... necessary to make a small but representative amount of pre-processed LSST data available in advance of each data release during Operations. (see (\S~\ref{sec:dmcon})."

I want to be careful with the word "release".  We will *need* to make available pre-processed data for training to the community who will train the estimators that provide photo-z as part of the DR. This does not necessarily mean we will make that data available to all DR holders. It doesn't mean we won't either;  but that decision should be left to operations.

File: DMTN-049.tex [hunk 8]
```diff
@@ -3,315 +3,628 @@
...
+If, however, the commissioning SV period is cut short, an {\it Early Science Period} of $\approx$3\,\textendash\,6 months that is different to regular survey operations may be enacted (Plan "B").
+During this period, it is envisaged that estimators that meet the criteria defined in \S~\ref{sec:eval} could continue to work on validation during this {\it Early Science Period}, and that a decision would be taken on which estimator(s) to run for DR1 prior to reverting to the LSST cadence and DR1. 
+If a further shakedown of operations procedures and data taking is still required at handover to Operations (Plan "C"), the implications for photo-$z$ estimators will be reviewed at that time based on the actual state of the system to determine the best course of action for photo-$z$ estimators. 
 
-In this section we demonstrate several analysis techniques for comparing the output of different photo-$z$ estimators, discussing each in turn below and then listing some potential future techniques in Section \ref{ssec:more}.
+In any of these scenarios,  estimators that will return the most accurate photo-$z$ as early in the survey as possible may be prioritized.
```
Comment (leannep) [inline DMTN-049.tex]: We said in the DM constraints section that "The strongest point of consideration for any contributed  photo-$z$ estimator will be its ability to run at LSST scale. " we also emphasized "robustness in the absence of fully representative training data" . This statement now seems inconsistent with that. Perhaps something like
" estimators that will return the most accurate photo-$z$ as early in the survey as possible and meet the criteria in  (\S~\ref{sec:dmcon}) ....."

File: DMTN-049.tex [hunk 9]
```diff
@@ -3,315 +3,628 @@
...
+
+\medskip
+\textbf{This roadmap is a living document} to guide this joint venture, and it will evolve over time to incorporate input from the science community.
+\textbf{The primary milestone in this roadmap is DM's call for "Letters of Recommendation" from the science community regarding photo-$z$ estimators (deadline 2021-09-30).}
+From these letters, DM will generate a shortlist of community-vetted photo-$z$ options that meet both the science community's needs for DR {\tt Object} photo-$z$ and the technical directives of the DM System.
+
+\medskip
+In support of this milestone, starting in early 2021, DM will host a series of "LSST Photo-z Virtual Forums", discussions open to all interested people, which will focus on defining the evaluation criteria for photo-$z$ estimators (\S~\ref{sec:eval}) and facilitating the development of "Letters of Recommendation" (\S~\ref{sec:lor}).
+Further on in this roadmap, as part of DM's process to select, implement, and validate one or more estimator(s) from the shortlist, DM will facilitate science community participation in a "Photo-$z$ Validation Cooperative" based on commissioning data (\S~\ref{sec:pzcoop}; dates TBD).
```
Comment (leannep) [inline DMTN-049.tex]: "as part of DM's mandate to compute photo-z , one or more .... "
(or remove completely)

File: DMTN-049.tex [hunk 10]
```diff
@@ -3,315 +3,628 @@
...
-\textbf{TPZ:} The results are quite poor 1 and 2 years, with a lot of quantization in the photo-$z$ and many outliers at low and high redshift, but are significantly improved at 10 years. It is very interesting that there is actually a large improvement if the training set does not have better photometric errors than the test set (i.e., compare the 1 and 2 year results in the third row to the second row), but this may just be related to how we've included the errors as \texttt{Attributes}. Either way, TPZ is sensitive to the provided training set, so an extended investigation into what would truly be a realistic 1 year spectroscopic training set for LSST should be done (e.g., different redshift distributions, different magnitude limits). Although TPZ appears to give better accuracy, we also need to ensure that it gives realistic precision for it's photo-$z$ results.
+\section{Appendix: LSST Documentation Review}\label{sec:docs}
```
Comment (leannep) [inline DMTN-049.tex]: Remove Appendix D and move D.4 DPDD to Appendix B.3, Outputs

**Review by leannep — CHANGES_REQUESTED** [review]

File: DMTN-049.tex (hunk 1 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Oh I didn't realize that was a standard, and the meaning of the slash confused me (in other contexts "/" is like "or" but in this case we mean Feb through Apr) -- anyway, changed back to 2021-02/04.

File: DMTN-049.tex (hunk 2 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Changed.

File: DMTN-049.tex (hunk 3 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: deleted

File: DMTN-049.tex (hunk 4 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: changed

File: DMTN-049.tex (hunk 5 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: added "if needed"

File: DMTN-049.tex (hunk 6 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Changed "Data Previews" --> "Preliminary LSST Data"

File: DMTN-049.tex (hunk 7 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Good catch, 'release' has a specific meaning and wasn't really appropriate here.

Comment (MelissaGraham) [reply DMTN-049.tex]: also I replaced the text with all your suggested wording

File: DMTN-049.tex (hunk 8 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Yes, good to cite S.3. as often as possible, too. Changed.

File: DMTN-049.tex (hunk 9 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: Went with "Further on in this roadmap DM will facilitate science community participation in a "Photo-$z$ Validation Cooperative" based on commissioning data for the shortlisted estimators (\S~\ref{sec:pzcoop}; dates TBD)."

File: DMTN-049.tex (hunk 10 above)
Comment (MelissaGraham) [reply DMTN-049.tex]: It is done, and all internal references to the DPDD definition of photo-z outputs have been re-directed to Appendix B.

**Review by leannep — APPROVED** [review]

**Pull request merged**